sdcm/sct_events.py

```diff
@@ -166,8 +166,10 @@ def __exit__(self, exception_type, exception_value, traceback):
         self.cancel_filter()
 
     def eval_filter(self, event):
-        name = self.type and self.type == getattr(event, 'type', '')
-        string = self.line and self.line in getattr(event, 'line', '')
+        line = getattr(event, 'line', '')
+        _type = getattr(event, 'type', '')
+        name = self.type and _type and self.type == _type
```

Comment:
name and string not meaningful
`name` -> `is_type_matching`
`string` -> `is_line_matching`

Reply:
fixed